gatsby-node.ts

```diff
@@ -434,6 +435,17 @@ export const onCreateWebpackConfig: GatsbyNode['onCreateWebpackConfig'] = ({
     return rule;
   });
 
+  if (config?.externals && config.externals[0]) {
+    if (
+      typeof config.externals[0] === 'object' &&
+      !Array.isArray(config.externals[0])
+    ) {
+      config.externals[0]['node:crypto'] = cryptoBrowserify;
+    }
+  }
+
+  actions.replaceWebpackConfig(config);
```

Comment:
Please merge with the `replaceWebpackConfig` call below.

Reply:
@Mrtenz basically delete this line, right?

Reply:
Even nicer would be to properly merge the two, and remove the first part. But this works for me.